### PR TITLE
[179445] Integrate Matomo

### PR DESCRIPTION
**Pull request by patryk-dabrowski**
[AB#179445](https://unicef.visualstudio.com/4e044e8d-bc28-4768-8074-f80ff6e4a0e5/_workitems/edit/179445)

### CODE REVIEW

File: backend/hct_mis_api/apps/administration/templates/admin/base.html [hunk 1]
```diff
@@ -1,22 +1,41 @@
...
+  {% if MATOMO_TRACKER_URL and MATOMO_SITE_ID %}
+    <!-- Matomo -->
+    <script>
+      var _paq = window._paq = window._paq || [];
+      /* tracker methods like "setCustomDimension" should be called before "trackPageView" */
+      _paq.push(['trackPageView']);
+      _paq.push(['enableLinkTracking']);
+      (function() {
+        var u = '{{ MATOMO_TRACKER_URL }}';
+        _paq.push(['setTrackerUrl', u + 'matomo.php']);
```
Comment (domdinicola) [inline backend/hct_mis_api/apps/administration/templates/admin/base.html:31]: remove mathomo.php

Comment (saxix) [reply backend/hct_mis_api/apps/administration/templates/admin/base.html:31]: why matomo in the admin ?

Comment (patryk-dabrowski) [reply backend/hct_mis_api/apps/administration/templates/admin/base.html]: @saxix related to the task `include the following script in HOPE pages (FE/React and Admin/Django template)`, matomo should be implemented also in the admin. Should I remove it from the admin if it should be implemented only in the HOPE pages

**Review by pavlo-mk — DISMISSED** [review]

File: backend/hct_mis_api/apps/web/templates/web/index.html [hunk 2]
```diff
@@ -21,6 +21,25 @@
     {% endif %}
   </script>
 
+  {% if MATOMO_TRACKER_URL and MATOMO_SITE_ID and MATOMO_SCRIPT_URL %}
```
Comment (wozniakpl) [inline backend/hct_mis_api/apps/web/templates/web/index.html:24]: can't this and the thing added in `backend/hct_mis_api/apps/administration/templates/admin/base.html` be a template and imported like this?

```
{% include "matomo.html" %}
```

Comment (patryk-dabrowski) [reply backend/hct_mis_api/apps/web/templates/web/index.html:24]: Good idea, I will change it 👍

**Review by wozniakpl — APPROVED** [review]

**Review by pavlo-mk — APPROVED** [review]

**Pull request merged**